data/tlg1443/tlg001/__cts__.xml

```diff
@@ -0,0 +1,8 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg1443" xml:lang="grc" urn="urn:cts:greekLit:tlg1443.tlg004">
+  <ti:title xml:lang="lat">Epistulae vii genuinae</ti:title>
```

Comment:
unchanged

Reply:
That is weird I changed it on my computer. I will try again.

Reply:
No wait now, I changed it from 004 to 001, that was what I wanted to do, the build is just not finished.